plugins/Kaleidoscope-Colormap-Overlay/src/kaleidoscope/plugin/Colormap-Overlay.h

```diff
@@ -58,7 +59,7 @@ class ColormapOverlay : public kaleidoscope::Plugin {
   // takes as its sole argument an array reference of size `_overlay_count`, so
   // there's no need to use `sizeof` to calculate the correct size, and pass it
   // as a separate parameter.
-  template<uint8_t _overlay_count>
+  template<uint8_t _overlay_count>  // TODO(EvyBongers): how to make sure that we don't exceed max_overlays_
```

Comment:
Similar question as above, though this is when building a sketch. Should I validate this to the maximum and throw an error?

Reply:
could you static assert so it throws a build error?